Libraries/WebSocket/__mocks__/event-target-shim.js

```diff
@@ -6,4 +6,13 @@
 // problem go away.
 
 'use strict';
-module.exports = function() {};
+
+function EventTarget() {
+  // Support both EventTarget and EventTarget([list, of, events])
+  // as a super class, just like the original module does.
+  if (arguments.length === 1 && Array.isArray(arguments[0])) {
```

Comment:
@philikon How does mocking the module this way work? Since `WebSocket` is calling methods like `this.dispatchEvent` on the base class, won't it throw error as they don't exist? I don't know the original reason on why it was written this way, probably some Jest issue reading from the comments.